src/frontend/src/components/public/CommentsSidebar/index.tsx

```diff
@@ -27,6 +30,12 @@ const CommentSidebar = ({
   const [modalTitleText, setModalTitleText] = useState<string>('');
   const [modalParagraph, setModalParagraph] = useState<string>('');
   const [modalButtonValue, setModalButtonValue] = useState<string>('');
+  const [enableComments, setEnableComments] = useState<boolean>(true);
+
+  useEffect(() => {
+    const allowComment = statusList(pia)[pia.status || 'complete'].comments;
```

Comment:
why should there be a default status here, and that too COMPLETE? I am assuming it was written just to get rid of the Linting error.  Added a suggestion, check if that makes more sense.

```suggestion
    if (!pia?.status) return;

    const allowComment = statusList(pia)?.[pia.status]?.comments || false;
```